.github/workflows/publish_snapshot.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   publish-snapshot:
-    if: github.repository == 'apache/orc'
+    if: ${{ github.repository == 'apache/orc' }}
```

Comment:
I know that the original code has no problem technically. 
I can give you the working reference. 

- https://github.com/apache/spark/blob/master/.github/workflows/publish_snapshot.yml#L9
- https://github.com/apache/spark/actions?query=workflow%3A%22Publish+Snapshot%22

So, please revise your PR description and revert this code change from this PR. You may try to file a new JIRA for this style fix.